src/cosmic_ray/operators/operator.py

```diff
@@ -13,12 +16,25 @@ class Operator(ABC):
 
     def __init__(self, python_version):
         self._python_version = python_version
+        self._pragma_category_name = None
 
     @property
     def python_version(self):
         "Python major.minor version as a string."
         return self._python_version
 
+    @property
+    def pragma_category_name(self):
```

Comment:
Pragmas have no place in the core CR code.

Reply:
That is precisely my note remark: This interceptor needs to know operators pragma-name, and it can be nice to configure this name per operator.
We can embed this name computations inside the interceptor, but there is no-way to have more user-friendly names.

Reply:
Surely the pragma interceptor can generate reasonable names from provider names coupled with the operator class names.